### PR TITLE
Chordgen - Sample Generator using Dynamic Random FX Processing

### PR DESCRIPTION
**Pull request by MortimerE**
Adds a modular signal processing effects rack to chordgen via samplegen.py 
Use for dataset synthesis and augmentation. This is currently decoupled from chordgen and wavgen which create midi and wavs respectively - samplegen takes an input folder of wav files and processes them.

### CODE REVIEW

**Comment by MortimerE** [comment]
Branch should be stable, check the changes and let me know if there's any fixes needed.

File: datagen/pedal_board.py [hunk 1]
```diff
@@ -4,100 +4,52 @@
 from scipy.signal import lfilter, butter
 import soundfile as sf
 import os
+from pedals import Compressor, Delay, Distortion, Reverb, Chorus, Noise, Flanger
 
 INPUT_PATH = "./wav_out/"
 OUTPUT_PATH = "./fx_out/"
```
Comment (maxwellherron5) [inline datagen/pedal_board.py:10]: total nitpick so feel free to disregard, but we should try to avoid relative paths

Comment (MortimerE) [reply datagen/pedal_board.py:10]: Agreed, will change these to use os later

**Comment by maxwellherron5** [comment]
Looks good and the documentation is great. I'm almost wondering if we want to place module specific documentation into the documents directory and call it `datagen.md`, just a thought though

**Review by zhutwo — APPROVED** [review]
gud

**Pull request merged**